interface/src/app/mlmodel/validator.tpl.html

```diff
@@ -0,0 +1,5 @@
+<div class="alert alert-danger" role="alert" ng-if="$ctrl.errMessage">
+  {{$ctrl.errMessage}}
+  <br>
+  Please click <a href="#/home">Home</a> to reset it.
```

Comment:
Can't change this now, but I see no reason to force the user to head back to the Home page to change the model. I'll file an issue to streamline this.